test/functional/fixtures/video-recording/test.js

```diff
@@ -0,0 +1,103 @@
+const expect = require('chai').expect;
+const config = require('../../config');
+const assertionHelper = require('../../assertion-helper.js');
+
+
+if (config.useLocalBrowsers) {
+    describe('Video Recording', () => {
+        afterEach(assertionHelper.removeVideosDir);
+
+        it('Should record video without options', () => {
+            return runTests('./testcafe-fixtures/index-test.js', '', {
+                only:         'chrome,firefox',
+                setVideoPath: true,
+                shouldFail:   true
+            })
+                .then(() => {
```

Comment:
We can remove this handler because we have handling of false-positive result in `runTests` method (if `shouldFail` is true), in this case we will get an error `new Error('Test should have failed but it succeeded');`